src/style.css

```diff
@@ -1,3 +1,78 @@
+:root {
+  --blue-color: rgba(0, 172, 220, 1); /* CSS-змінна для синього кольору */
+}
+
 body {
+  padding: 0;
+  box-sizing: border-box;
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  box-shadow: 0 2px 5px 0 rgba(0, 0, 0, 0.18);
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.box {
+  text-align: center;
+  position: relative;
+  font-size: 12px;
+  text-decoration: none;
+  color: black;
+  height: 60px; /* Висота посилання */
+  line-height: 60px; /* Вирівнювання тексту вертикально */
```

Comment:
The height is correctly set for the links, which aligns with the task requirements. Ensure that this is consistently applied across all relevant elements.